geode-assembly/src/acceptanceTest/java/org/apache/geode/client/sni/ClientSNIAcceptanceTest.java

```diff
@@ -87,7 +87,7 @@ public void connectToSNIProxyDocker() {
         .port(15443)
         .getExternalPort();
     ClientCache cache = new ClientCacheFactory(gemFireProps)
-        .addPoolLocator("locator", 10334)
+        .addPoolLocator("locator-maeve", 10334)
```

Comment:
I would not have spelled that name correctly, kudos William

Reply:
🤠